project.clj

```diff
@@ -105,7 +106,7 @@
                        :build-type "foss"
                        :main-namespace "puppetlabs.puppetdb.main"
                        :create-varlib true}
-                :config-dir "ext/config"
+                :config-dir "ext/config/foss"
```

Comment:
What's `:config-dir` used for? If were not using it @waynr should it be an optional are to ezbake?